docs/articles/step-5-project-sharing.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Step 5 - Adding project collaborators
+---
+
+In this guide, we'll walk you through the steps to add a new user to your
+account and share the project with them.
+
+A project can only be shared if the account plan allows for multiple users and
```

Comment:
I couldn't find docs on the number of users allowed per plan, do we have that or should I update those too ?